services/src/main/java/org/keycloak/protocol/oid4vc/issuance/OID4VCIssuerEndpoint.java

```diff
@@ -384,6 +393,7 @@ public Response requestCredential(
         Object theCredential = getCredential(authResult, supportedCredentialConfiguration, credentialRequestVO);
         if (SUPPORTED_FORMATS.contains(requestedFormat)) {
             responseVO.setCredential(theCredential);
+            responseVO.setNotificationId(generateNotificationId());
```

Comment:
This alternative syntax should be possible.
```suggestion
            responseVO
                .setCredential(theCredential)
                .setNotificationId(generateNotificationId());
```